packages/hooks-react/src/useSearchQueryUpdater.ts

```diff
@@ -4,6 +4,8 @@ import { useUIStore } from '@jwp/ott-common/src/stores/UIStore';
 
 import useDebounce from './useDebounce';
 
+// @todo: should move entirely to ui-react?
```

Comment:
We can also add a `onChange` argument so we can move the `updateSearchPath` function to the Search page (implementation). This will remove the react-router dependency.

But ... do we need this component for other platforms as well?

Reply:
I'm moving it from my next PR (cleaning `react-router` from `hooks-react`). I don't see the point in making this reusable, as it is only used by `ui-react`.